src/handlers/payout/index.ts

```diff
@@ -34,8 +35,17 @@ export const handleIssueClosed = async () => {
   console.log({ assignee });
   const recipient = await getWalletAddress(assignee.login);
   if (!recipient) {
-    log.info(`Recipient address is missing`);
-    await addCommentToIssue(`@${assignee.login} would you please post your wallet address here?`, issue.number);
+    const {
+      data: { state_reason },
+    } = await context.octokit.issues.get({
+      owner: payload.organization!.login,
+      repo: payload.repository.name,
+      issue_number: issue.number,
+    });
```

Comment:
We have a payload including issue object. Please take a look at the payload object and getBotContext(). We shouldn't get issue object again in the codebase, should use the one provided by Probot (basically both are same)

Reply:
the issue object doesn't have the `state_reason` property